packages/scandipwa/src/component/CategoryConfigurableAttributes/CategoryConfigurableAttributes.container.js

```diff
@@ -19,7 +19,8 @@ import CategoryConfigurableAttributes from './CategoryConfigurableAttributes.com
 /** @namespace Component/CategoryConfigurableAttributes/Container/mapStateToProps */
 export const mapStateToProps = (state) => ({
     currency_code: state.ConfigReducer.currencyData.current_currency_code,
-    show_product_count: state.ConfigReducer.layered_navigation_product_count_enabled
+    show_product_count: state.ConfigReducer.layered_navigation_product_count_enabled,
+    childrenCategories: state.CategoryReducer.category.children
```

Comment:
I see that you have added a new prop to container, that is assigned to the component where you are using it.
But i don't see that you define the new prop in the prop types.
This should be required by the eslint. So either you don't have eslint plugin enabled in your IDE or eslint rule got broken.
Anyway, please add the new prop in the component's prop types object.

Reply:
I still don't see proptypes for this.